legal-api/report-templates/template-parts/incorporation-application/incorporationDetails.html

```diff
@@ -4,18 +4,25 @@
             <tr>
                <td>
                   <table class="business-info" role="presentation">
+                     {% if header.reportType == 'noa'%}
+                     <tr>
+                        <td class="section-sub-title">Incorporation Number:</td>
+                        <td>{{business.identifier}}</td>
```

Comment:
Is this value declared before the IA is completed? I'm wondering if this will look OK when a future effective IA is filed (PAID) but not yet completed?

Reply:
There is a condition check for noa - this section will be evaluated only for Notice of Articles which will be available only after the filing is complete